jam/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       APP_PORT: 80
 
   jam:
-    image: ghcr.io/joinmarket-webui/joinmarket-webui-standalone:v0.0.9-clientserver-v0.9.6@sha256:f36da8ed2f75b8db8dca34783b1e1ed11d0048270140b1c2cc072b15191e6b6c
+    image: ghcr.io/joinmarket-webui/joinmarket-webui-standalone:v0.0.9-clientserver-v0.9.6@sha256:4287ebb5d7d9bda7acdc77662df7adf0b6e7a5974a39a314c3ec0e85336ec1d5
```

Comment:
The digest for a multi-arch image should be `sha256:f36da8ed2f75b8db8dca34783b1e1ed11d0048270140b1c2cc072b15191e6b6c` here.

Reply:
Oh man, I forgot to adapt the version : /
Well.. that's embarrassing. Fix now.  Thank you.